sdk/keyvault/azsecrets/client.go

```diff
@@ -397,12 +395,29 @@ func (c *Client) GetDeletedSecret(ctx context.Context, secretName string, option
 type UpdateSecretPropertiesOptions struct {
 	// Version is the specific version of a Secret to update. If not specified it will update the most recent version.
 	Version string
+
+	// Type of the secret value such as a password.
+	ContentType *string `json:"contentType,omitempty"`
+
+	// The secret management attributes.
+	SecretAttributes *Properties `json:"attributes,omitempty"`
```

Comment:
`SecretProperties` (won't repeat, so please check all occurrences).

Reply:
Back to the stuttering discussion from other PRs, I'm going to keep this as `Properties`.